similarity/src/main/java/edu/illinois/cs/cogcomp/sim/WordSim.java

```diff
@@ -0,0 +1,98 @@
+package edu.illinois.cs.cogcomp.sim;
```

Comment:
This class needs to change.  It is unlikely anyone will want to instantiate every similarity metric simultaneously in a single object. The Embeddings, and MemoryBasedESA could all implement a common interface, then WordSim can take one of these as a constructor argument and call its `compare()` (or whatever) method regardless of its type.